packages/ui-voip/src/views/CallHistoryContextualbar/CallHistoryContextualbar.tsx

```diff
@@ -75,7 +75,7 @@ const CallHistoryContextualBar = ({ onClose, actions, contact, data }: CallHisto
 			</ContextualbarHeader>
 			<ContextualbarScrollableContent>
 				<InfoPanel>
-					<InfoPanelSection>
+					<InfoPanelSection fontScale='p1b'>
```

Comment:
I this component shouldn't have a font variation. I can't imagine why we have a component to define a font and then we change it.

Reply:
Initially I was changing this in the subcomponent but that one is reused in other places. I understand the concern but I argue that we should not expose styling props if we should never change them...

Still, we can always "hack" a way to change styles by modifying subcomponents, if I had done that we would have the same "issue" at hand, but it was probably going to be unnoticed if I had done it that way.

Reply:
`type InfoPanelSectionProps = ComponentPropsWithoutRef<typeof Box>;`

Bad component design leads to bad component usage.